### DISCUSSION TITLE
wtf academy 在discord 无法加入

### DISCUSSION THREAD

**Discussion by SolZm** [AmazingAng/WTF-Solidity#471] [General]
wtf academy 在discord 无法加入

**Comment by Azleal** [comment]
> wtf academy 在discord 无法加入

https://discord.gg/SF43d6kNQF

**Reply by AmazingAng** [reply]
这个一般是网络问题

**Comment by SolZm** [comment]
点击加入，会出现这个提示，然后加不进去

[image: 14391681116096_.pic.jpg]

Azleal ***@***.***> 于2023年4月10日周一 16:24写道：

**Comment by XdpCs** [comment]
https://discord.gg/5akcruXrsk
